src/screens/Authentication/AuthHeader.tsx

```diff
@@ -0,0 +1,47 @@
+import { WithTooltip } from "@storybook/components";
+import { ChevronLeftIcon, QuestionIcon } from "@storybook/icons";
+import { styled } from "@storybook/theming";
+import React from "react";
+
+import { IconButton } from "../../components/IconButton";
+import { Bar, Col } from "../../components/layout";
+import { TooltipNote } from "../../components/TooltipNote";
+
+const HeaderButton = styled(IconButton)(({ theme }) => ({
+  color: theme.base === "light" ? "currentColor" : theme.color.medium,
+  fontSize: theme.typography.size.s2 - 1,
```

Comment:
Thanks for following the designs, but Storybook's UI now mostly uses 14px and 12px. Can you remove `- 1` and make the `fontWeight: theme.typography.weight.bold`? Other than that, everything else looks good to go.

Reply:
Done 👍